src/components/pages/charactersPage.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import CharactersProfile from '../charactersProfile';
+
+const CharactersPage = () => (
```

Comment:
- [ ] Once more, kindly check out the differences in design from our design guidelines and equally make the necessary changes on this page(CharactersPage)